ivy_tests/test_ivy/test_frontends/test_torch/test_blas_and_lapack_ops.py

```diff
@@ -848,37 +849,62 @@ def test_torch_qr(
 @handle_frontend_test(
     fn_tree="torch.svd",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float", index=1),
-        min_num_dims=3,
-        max_num_dims=5,
-        min_dim_size=2,
-        max_dim_size=5,
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=0,
+        max_value=10,
+        shape=helpers.ints(min_value=2, max_value=5).map(lambda x: (x, x)),
     ),
     some=st.booleans(),
-    compute=st.booleans(),
+    compute_uv=st.booleans(),
 )
 def test_torch_svd(
     dtype_and_x,
     some,
-    compute,
-    on_device,
-    fn_tree,
+    compute_uv,
     frontend,
     test_flags,
+    fn_tree,
     backend_fw,
+    on_device,
 ):
-    dtype, x = dtype_and_x
-    helpers.test_frontend_function(
-        input_dtypes=dtype,
+    input_dtype, x = dtype_and_x
+    x = np.asarray(x[0], dtype=input_dtype[0])
+    # make symmetric positive definite
+    x = np.matmul(x.T, x) + np.identity(x.shape[0]) * 1e-3
+    ret, frontend_ret = helpers.test_frontend_function(
+        input_dtypes=input_dtype,
         backend_to_test=backend_fw,
         frontend=frontend,
         test_flags=test_flags,
         fn_tree=fn_tree,
         on_device=on_device,
-        input=x[0],
+        test_values=False,
+        input = x,
         some=some,
-        compute_uv=compute,
+        compute_uv=compute_uv,
     )
+    ret = [np.asarray(x) for x in ret]
```

Comment:
```suggestion
    if backend_fw == "torch":
        ret = [x.detach() for x in ret]
    ret = [np.asarray(x) for x in ret]
```
You can fix the error you're seeing with something like this. You'll need to implement this for the other tests as well.

Reply:
Thanks a lot!